### ISSUE TITLE
Fix yet another JavaScript regex issue in `every_javascript_string`

### ISSUE THREAD

**Issue by postmodern** [ronin-rb/ronin-web-spider#28]
Appears that this inline JavaScript regex does not match our regex for inline JavaScript regexes.

```javascript
/^(?:https?://)?(?:www|player.)?(?:vimeo.com/)?(?:video/|external/)?(d+)([^.?&#"'>]?)/
```

This then causes the JavaScript string scanner to not properly skip over the entire inline regex.

**Comment by postmodern** [comment]
Ah, it's because we match the code before the inline regex, and the JavaScript code in question has no spaces:

```javascript
getURLRegex(){return/^(?:https?:\/\/)?(?:www|player\.)?(?:vimeo\.com\/)?(?:video\/|external\/)?(\d+)([^.?&#"'>]?)/}
```

**Comment by postmodern** [comment]
Fixed by 6fa6ca0.
